### ISSUE TITLE
🐛 [Bug] Encountered bug when using TRTorch 0.3.0 and torch_1.6.0_update versions on Jetson Xavier AGX

### ISSUE THREAD

**Issue by OronG13** [NVIDIA/TRTorch#171]
## ❓ Question

Are there some missing instructions on how to build the TRTorch 'torch_1.6.0_update' for Jetson Xavier AGX?

## What you have already tried

- Downloaded two TRTorch versions: 0.3.0. tag and torch_1.6.0_update

- Follow the Compilations TRTorch instructions of both versions

- Both versions compilation have errors:

    - 0.3.0: 
       core/lowering/lowering.cpp:10:10: fatal error: torch/csrc/jit/passes/quantization.h: No such file or directory
       #include "torch/csrc/jit/passes/quantization.h"

     - torch_1.6.0_update:

         ERROR: 
        /home/ubuntu/.cache/bazel/_bazel_root/7f0ba44765888be019f9da1ca19341ed/external/tensorrt/BUILD.bazel:63:10: 
       @tensorrt//:nvinfer_lib: invalid label '' in each branch in select expression of attribute 'static_library' in 'cc_import' rule 
       (including '//conditions:default'): empty package-relative label
       ERROR: /home/ubuntu/Downloads/TRTorch-torch_1.6.0_update/core/util/BUILD:69:11: Target '@tensorrt//:nvinfer' contains 
       an error and its package is in error and referenced by '//core/util:trt_util'
       ERROR: Analysis of target '//:libtrtorch' failed; build aborted: Analysis failed

## Environment
> Jetson Xavier AGX with JetPack 4.4 

 - PyTorch Version (e.g., 1.0): 1.6.0
 - CPU Architecture: Jetson Xavier AGX
 - OS (e.g., Linux): Jetson Xavier AGX JetPack 4.4
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source): **pip**
 - Build command you used (if compiling from source): NA
 - Are you using local sources or building from archives: Build from JetPack 4.4
 - Python version: 3.6.9
 - CUDA version: 10.2
 - GPU models and configuration: Jetson Xavier AGX
 - Any other relevant information:

## Additional context
Attached are my edited WORKSPACE files.

[TRTorchBuildErrors.zip](https://github.com/NVIDIA/TRTorch/files/5068075/TRTorchBuildErrors.zip)

**Comment by narendasan** [comment]
Version 0.3.0 will not work with PyTorch 1.6.0. You will need to use the update branch which is still in progress. For now if possible I recommend using PyTorch 1.5.1

**Comment by OronG13** [comment]
Thanks for the clarification,
I tried to install PyTorch 1.5.1 on my Jetson Xavier AGX unsuccessfully:

- Tried to find a built binaries in order to install it using the pip3 command but based on this link it isn't exit:
[https://forums.developer.nvidia.com/t/pytorch-for-jetson-nano-version-1-6-0-now-available/72048](url)

- Tried to build from source the v1.5.1 Tag from here:
[https://github.com/pytorch/pytorch/tree/v1.5.1](url)
But some compilations errors are notified.

When I tried to build this PyTorch tag on my Linux desktop it was completed OK.
I tried to build a simple C++ tester which trying to load a model (that was serialized from Python interface using the torch.jit.trace) using the torch::jit::load C++ API but I got some run time exception 
(A problem report was opened by me in the PyTorch:
[https://discuss.pytorch.org/t/a-simple-pytorch-c-tester-has-linker-errors/92562](url) ).

Based on the examples here (In section **Compiling with TRTorch in C++** ):
[https://nvidia.github.io/TRTorch/tutorials/getting_started.html](url)

I understand that before I will be able to use TRTorch C++ APIs I must load the mode using the torch::jit::load API.
Am I right?

Regard,
